src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getLinkDataCy(index: number, totalPages: number): string {
+  if (index === 0) {
+    return 'prevLink';
+  } else {
+    return index === totalPages + 1 ? 'nextLink' : 'pageLink';
```

Comment:
Consider using zero-based indexing for `totalPages`. This would align the page indexing with common programming practices and avoid the need to add 1. If you decide to stick with the current approach, make sure to document the reason for using one-based indexing for clarity.